internal/server/middleware/gziper.go

```diff
@@ -0,0 +1,27 @@
+package middleware
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"strings"
+)
+
+var gzipSkipper = func(c echo.Context) bool {
+	accept := c.Request().Header.Get(echo.HeaderAcceptEncoding)
+	hasNoGzip := !strings.Contains(accept, "gzip")
+	if !hasNoGzip {
+		c.Response().Header().Set(echo.HeaderContentEncoding, "gzip")
+	}
+	return hasNoGzip
+}
+
+func newGzipConfig(f func(c echo.Context) bool) middleware.GzipConfig {
```

Comment:
You need to use `typename` for `func(c echo.Context) bool`

Reply:
done